EESSI-pilot-install-software.sh

```diff
@@ -142,94 +142,65 @@ else
-$EB --from-pr 11368 makeinfo-6.7-GCCcore-9.3.0.eb --robot && $EB --from-pr 11454 DB-18.1.32-GCCcore-9.3.0.eb --robot && $EB --from-pr 11200 --robot
-if [[ $? -eq 0 ]]; then
-    echo_green "${PERL_EC} installed via easyconfigs PR #11200, that was just a small side step, don't worry..."
-else
-    error "Installation of ${PERL_EC} failed!"
-fi
-
-# side-step to fix installation of CMake with zlib included in --filter-deps
-# see https://github.com/easybuilders/easybuild-easyblocks/pull/2187
-echo ">> Installing CMake with fixed easyblock..."
-$EB CMake-3.16.4-GCCcore-9.3.0.eb --include-easyblocks-from-pr 2187 --robot
-if [[ $? -eq 0 ]]; then
-    echo_green "CMake installation done, glad that worked out!"
-else
-    error "Installation of CMake failed, pfft..."
-fi
-
-# required to make sure that libraries like zlib that are listed in --filter-deps can be found by pkg-config
-# FIXME: fix this in EasyBuild framework!
-#        see https://github.com/easybuilders/easybuild-framework/pull/3451
-export PKG_CONFIG_PATH=$EPREFIX/usr/lib64/pkgconfig
-
-# FIXME custom installation of Qt5 with patch required to build with Gentoo's zlib
-# see https://github.com/easybuilders/easybuild-easyconfigs/pull/11385
-echo ">> Installing Qt5 with extra patch to use zlib provided by Gentoo..."
-$EB --from-pr 11385 --robot
-if [[ $? -eq 0 ]]; then
-    echo_green "Done with custom Qt5!"
-else
-    error "Installation of custom Qt5 failed, grrr..."
-fi
-
-echo ">> Installing OpenBLAS, Python 3 and Qt5..."
 # If we're building OpenBLAS for GENERIC, we need https://github.com/easybuilders/easybuild-easyblocks/pull/1946
+echo ">> Installing OpenBLAS..."
 if [[ $GENERIC -eq 1 ]]; then
     echo_yellow ">> Using https://github.com/easybuilders/easybuild-easyblocks/pull/1946 to build generic OpenBLAS."
-    $EB --include-easyblocks-from-pr 1946 OpenBLAS-0.3.9-GCC-9.3.0.eb Python-3.8.2-GCCcore-9.3.0.eb Qt5-5.14.1-GCCcore-9.3.0.eb --robot
+    $EB --include-easyblocks-from-pr 1946 OpenBLAS-0.3.9-GCC-9.3.0.eb --robot
 else
-    $EB OpenBLAS-0.3.9-GCC-9.3.0.eb Python-3.8.2-GCCcore-9.3.0.eb Qt5-5.14.1-GCCcore-9.3.0.eb --robot
+    $EB OpenBLAS-0.3.9-GCC-9.3.0.eb --robot
 fi
 if [[ $? -eq 0 ]]; then
-    echo_green "Done with OpenBLAS, Python 3 and Qt5!"
+    echo_green "Done with OpenBLAS!"
 else
-    error "Installation of OpenBLAS, Python 3 and Qt5 failed!"
+    error "Installation of OpenBLAS failed!"
 fi
 
-# FIXME: customized installation of OpenMPI, that supports high speed interconnects properly...
-#        see https://github.com/EESSI/software-layer/issues/14
-echo ">> Installing properly configured OpenMPI..."
-$EB --from-pr 11387 OpenMPI-4.0.3-GCC-9.3.0.eb --include-easyblocks-from-pr 2188 --robot
+echo ">> Installing OpenMPI..."
+$EB OpenMPI-4.0.3-GCC-9.3.0.eb --robot
 if [[ $? -eq 0 ]]; then
     echo_green "OpenMPI installed, w00!"
 else
     error "Installation of OpenMPI failed, that's not good..."
 fi
 
```

Comment:
```suggestion

# FIXME: fix this in Easybuild by making a template value for sysroot available to easyconfigs and using that in all fontconfig easyconfigs
# for now, we use a custom easyconfig that sets ${EPREFIX}/usr/share/fonts as an additional fonts directory in configopts
echo ">> Installing fontconfig..."
$EB ./fontconfig-2.13.92-GCCcore-9.3.0.eb --robot
if [[ $? -eq 0 ]]; then
    echo_green "Done with fontconfig!"
else
    error "Installation of fontconfig failed!"
fi

```

Use something like this for fontconfig?

Reply:
I wonder if we can do better, since this way you need to be aware that you need to copy both the script and the custom easyconfig(s?)...

Should we try downloading it from the repo on the fly in the script?

Reply:
Yeah, maybe that's better.